docs/spark_custom_resources.md

```diff
@@ -0,0 +1,203 @@
+## Spark Operator API
```

Comment:
License header?

Reply:
Thanks for the catch! 

We have disabled license header check for markdowns, but I added this back for files under `/docs` for consistency